components/prism-arturo.js

```diff
@@ -0,0 +1,99 @@
+/**
+ * @param {string} lang
+ * @param {string} pattern
+ */
+ function createLanguageString(lang, pattern) {
```

Comment:
Right, I forgot to mention something. People typically import these language files directly, so this function is leaked into global scope. Please wrap the whole file into an IIFE. [Like this](https://github.com/PrismJS/prism/blob/d17e37629a9daa31d4a20ab4a282f54e42f8869b/components/prism-java.js#L1).

Reply:
Done 😉